homeassistant/components/pi_hole/__init__.py

```diff
@@ -1 +1,107 @@
 """The pi_hole component."""
+import logging
+import voluptuous as vol
+
+from homeassistant.const import (
+    CONF_HOST,
+    CONF_NAME,
+    CONF_MONITORED_CONDITIONS,
+    CONF_SSL,
+    CONF_VERIFY_SSL,
+)
+from homeassistant.components.sensor import DOMAIN as SENSOR_DOMAIN
+from homeassistant.helpers import config_validation as cv
+from homeassistant.exceptions import PlatformNotReady
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers.discovery import async_load_platform
+from homeassistant.util import Throttle
+
+from .const import (
+    DOMAIN,
+    CONF_LOCATION,
+    DEFAULT_HOST,
+    DEFAULT_LOCATION,
+    DEFAULT_NAME,
+    DEFAULT_SSL,
+    DEFAULT_VERIFY_SSL,
+    MIN_TIME_BETWEEN_UPDATES,
+    MONITORED_CONDITIONS,
+)
+
+LOGGER = logging.getLogger(__name__)
+
+CONFIG_SCHEMA = vol.Schema(
+    {
+        DOMAIN: vol.Schema(
+            {
+                vol.Optional(CONF_HOST, default=DEFAULT_HOST): cv.string,
+                vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+                vol.Optional(CONF_SSL, default=DEFAULT_SSL): cv.boolean,
+                vol.Optional(CONF_LOCATION, default=DEFAULT_LOCATION): cv.string,
+                vol.Optional(CONF_VERIFY_SSL, default=DEFAULT_VERIFY_SSL): cv.boolean,
+                vol.Optional(
```

Comment:
We should drop monitored conditions. All entities should be added to the state machine.